prime-router/src/main/resources/db/migration/V57__update_livd_materialized_view.sql

```diff
@@ -0,0 +1,115 @@
+     , r.data->>'Vendor Reference ID'                 as "vendor_reference_id"
+     , r.data->>'Testkit Name ID'                     as "testkit_name_id"
+     , r.data->>'Testkit Name ID Type'                as "testkit_name_id_type"
+     , r.data->>'Equipment UID'                       as "equipment_uid"
+     , r.data->>'Equipment UID Type'                  as "equipment_uid_type"
+     , r.data->>'Component'                           as "component"
+     , r.data->>'Property'                            as "property"
+     , r.data->>'Time'                                as "time"
+     , r.data->>'System'                              as "system"
+     , r.data->>'Scale'                               as "scale"
+     , r.data->>'Method'                              as "method"
+     , r.data->>'Publication Version ID'              as "publication_version_id"
+     , r.data->>'LOINC Version ID'                    as "loinc_version_id"
+     , r.data->>'Saliva'                              as "saliva"
+     , r.data->>'Pooling'                             as "pooling"
+     , r.data->>'Screening'                           as "screening"
+     , r.data->>'Test Type'                           as "test_type"
+     , r.data->>'CLIAComplexity'                      as "clia_complexity"
+     , r.data->>'Fingerstick'                         as "fingerstick"
+     , r.data->>'Test Method'                         as "test_method"
+     , r.data->>'Multi-analyte'                       as "multi-analyte"
+     , r.data->>'Collection Kit'                      as "collection_kit"
+     , r.data->>'Serial Testing'                      as "serial_testing"
+     , r.data->>'Home Collection'                     as "home_collection"
+     , r.data->>'Serial Screening'                    as "serial_screening"
+     , r.data->>'Authorized Settings'                 as "authorized_settings"
+     , r.data->>'Quantitative Result'                 as "quantitative_result"
+     , r.data->>'Direct to Consumer (DTC)'            as "direct_to_consumer"
+     , r.data->>'Non-prescription Testing'            as "non-prescription_testing"
+     , r.data->>'Prescription Home Testing'           as "prescription_home_testing"
+     , r.data->>'Visual or Instrument Read'           as "visual_or_instrument_read"
+     , r.data->>'Show All Home Testing Rows'          as "show_all_home_testing_rows"
+     , r.data->>'Single or Multiple Targets'          as "single_or_multiple_targets"
+     , r.data->>'Telehealth Proctor Supervised'       as "telehealth_proctor_supervised"
+     , r.data->>'Pooled Serial Screening - Swab'      as "Pooled_serial_screening_swab"
+     , r.data->>'Pooled Serial Screening - Media'     as "pooled_serial_screening_media"
+     , r.data->>'Over the Counter (OTC) Home Testing' as "over_the_counter_home_testing"
+     -- additional fields we add for our own needs
+     , r.data->>'is_otc'                              as "is_otc"
```

Comment:
Do we still need all these fields from the supplemental table if we are bringing in the extra fields from the new livid version? It looks like we might have test devices in the supplemental table that are used for integration tests though.